### PR TITLE
Specify img width.

### LINKED ISSUE #268: Fix 1 px image and summary pick up.

### PR DESCRIPTION
**Pull request by alexgerstein**
Closes #268.

### CODE REVIEW

File: worker/scraper.py [hunk 1]
```diff
@@ -49,12 +49,12 @@ def scrape_feed_articles(feed):
         # Extract first image from item
         try:
             summary_soup = BeautifulSoup(item.summary)
-            img_url = summary_soup.find('img')['src']
+            img_url = summary_soup.find('img', width_=lambda x: x != 1)['src']
```
Comment (swglad) [inline worker/scraper.py]: Does this solve the problem (e.g. on the Techcrunch item cards)?

Comment (alexgerstein) [inline worker/scraper.py]: Yeah. It fixes the issue of the 1px items becoming the card thumbnails.

Comment (swglad) [inline worker/scraper.py]: Ah, very good.

Did you work on the other part the issue: to make sure that the 1px images inside the cards don't stretch out and appear to add "blank" space?

File: sourcemash/frontend/static/scss/sourcemash.scss [hunk 2]
```diff
@@ -366,6 +366,10 @@ a.button-collapse.top-nav {
     .activator {
         cursor: pointer;
     }
+    img[width*="1px"] {
```
Comment (swglad) [inline sourcemash/frontend/static/scss/sourcemash.scss]: Should you add this for height=1px too?

Comment (alexgerstein) [inline sourcemash/frontend/static/scss/sourcemash.scss]: I don't think thats necessary

Alex Gerstein
gerstein.alex@gmail.com




On Sun, May 17, 2015 at 4:28 PM -0700, "Scott Gladstone" <notifications@github.com> wrote:












In sourcemash/frontend/static/scss/sourcemash.scss:
> @@ -366,6 +366,10 @@ a.button-collapse.top-nav {
>      .activator {
>          cursor: pointer;
>      }
> +    img[width*="1px"] {



Should you add this for height=1px too?

**Pull request merged**